### PR TITLE
Add and remove events from productions

### PR DESCRIPTION
**Pull request by jdrieghe**
### Added
- Command to add event to an existing production
- Command to remove event from an existing production

---
Ticket: https://jira.uitdatabank.be/browse/III-3324

### CODE REVIEW

File: tests/Event/Productions/ProductionCommandHandlerTest.php [hunk 1]
```diff
@@ -97,4 +97,30 @@ public function itCannotAddAnEventThatAlreadyBelongsToAnotherProduction():void
             new AddEventToProduction($eventBelongingToFirstProduction, $secondProductionCommand->getProductionId())
         );
     }
+
+    /**
+     * @test
+     */
+    public function itCanRemoveAnEventFromAProduction(): void
```
Comment (konrad126) [inline tests/Event/Productions/ProductionCommandHandlerTest.php:104]: Could you switch it to snake_case to be consistent with other tests (or is there a reason for using camelCase here specifically?)

Comment (jdrieghe) [reply tests/Event/Productions/ProductionCommandHandlerTest.php:104]: No specific reason other than muscle memory. I'll snake case the tests here

Comment (jdrieghe) [reply tests/Event/Productions/ProductionCommandHandlerTest.php:104]: You know what, I'll do it in the follow-up PR because that one adds another test.

Comment (konrad126) [reply tests/Event/Productions/ProductionCommandHandlerTest.php:104]: Yeah ofc, it's no biggie :)

Comment (jdrieghe) [reply tests/Event/Productions/ProductionCommandHandlerTest.php:104]: Fixed here: https://github.com/cultuurnet/udb3-silex/pull/465/commits/72912f14550753adf5cdc2168ebc2a4c52c3be82

**Pull request merged**